docker/puppetserver-base/Dockerfile

```diff
@@ -64,4 +64,4 @@ RUN chmod +x /docker-entrypoint.sh /healthcheck.sh && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-COPY Dockerfile /
+COPY Dockerfile /base.Dockerfile
```

Comment:
We might be able to get rid of the base file completely later, if we use `--target` along with `DOCKER_BUILDKIT=1`... so this may ultimately not be too important. But makes sense to do this for now IMHO.